cluster/config-defaults.yaml

```diff
@@ -119,6 +119,15 @@ cadvisor_memory: "150Mi"
 node_exporter_cpu: "20m"
 node_exporter_memory: "75Mi"
 
+# journald reader settings
+{{if eq .Environment "e2e"}}
+journald_reader_enabled: "true"
+{{else}}
+journald_reader_enabled: "false"
+{{end}}
+journald_reader_cpu: "1m"
+journald_reader_memory: "75Mi"
```

Comment:
Isn't that too much?

Reply:
I'll reduce it later after figuring out how much it actually needs.